compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/input/key/KeyEvent.web.kt

```diff
@@ -41,16 +29,26 @@ private fun KeyboardEvent.toInputModifiers(): PointerKeyboardModifiers {
     )
 }
 
+private fun KeyboardEvent.resolveCodePoint(composeKey: Key): Int {
+    val isMeta = when (composeKey) {
+        Key.MetaLeft, Key.MetaRight, Key.AltLeft, Key.AltRight, Key.CtrlLeft, Key.CtrlRight -> true
+        else -> false
+    }
+
+    return if (key == code || isMeta) composeKey.keyCode.toInt() else key.codePointAt(0)
```

Comment:
I'm not sure if `key == code` can be true at all - https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code

For example when pressing 'J' key:  KeyboardEvent: key='j' | code='KeyJ' 
Or Shift + J: KeyboardEvent: key='J' | code='KeyJ'

When would it be true?

Reply:
@eymar  those are covered in tests - namely, pairs like "Backspace"/ "Backspace", "Delete"/"Delete" etc.
however!!!!

after giving it some thought here are my doubts about this approach - 
we already have - and actually don't giving it up in the limits of this PR - check for key[0] == key - it's used in `isTypedEvent`. The purpose of this changes is quite similar - we want KeyEvent to pass actual code points. 
So what do you think about having exactly that kind of check here as well. 

that is, something like this:
`if (isMeta || key.firstOrNull()?.toString() != key)` compose.keyCode.toInt() else key.codePointAt(0)`

what do you think?